bciers/apps/compliance/src/tests/components/compliance-summaries/ComplianceSummariesPage.test.tsx

```diff
@@ -73,13 +104,89 @@ describe("ComplianceSummariesPage", () => {
     expect(fetchSpy).toHaveBeenCalledWith(mockSearchParams);
   });
 
-  it("renders alert message with dynamic reporting year", async () => {
+  // neither alerts
+  it("doesn't render either alert messages if no obligations fully met", async () => {
+    fetchSpy.mockResolvedValue(mockFetchResponseNoUnmetObligations);
+    await renderPage();
+
+    expect(
+      screen.queryByText(
+        /Your compliance obligation for the 2024 reporting year is/i,
+      ),
+    ).toBeNull();
+    expect(
+      screen.queryByText(/An automatic overdue penalty has been incurred/i),
+    ).toBeNull();
+  });
+
+  // first alert only
+  it("renders 1st alert message with dynamic reporting year", async () => {
+    const currentDate = new Date("2025-10-01");
+    vi.setSystemTime(currentDate);
     await renderPage();
     expect(
       screen.getByText(
         /Your compliance obligation for the 2024 reporting year is/i,
       ),
     ).toBeVisible();
     expect(screen.getByText(/due on November 30, 2025/i)).toBeVisible();
+    expect(
+      screen.queryByText(/An automatic overdue penalty has been incurred/i),
+    ).toBeNull();
+  });
+
+  // first alert modified
+  it("renders 1st alert message with 'was' if current date is past due", async () => {
+    fetchSpy.mockResolvedValue(mockFetchResponse);
+    const currentDate = new Date(2025, 11, 1);
+    vi.setSystemTime(currentDate);
+    await renderPage();
+    expect(
+      screen.getByText(
+        /Your compliance obligation for the 2024 reporting year was/i,
+      ),
+    ).toBeVisible();
+    expect(screen.getByText(/due on November 30, 2025/i)).toBeVisible();
+  });
+
+  // both alerts
+  it("renders both alert messages for overdue penalty if date was past due and initial data has obligation not met", async () => {
+    const currentDate = new Date(2025, 11, 1);
+    vi.setSystemTime(currentDate);
+
+    await renderPage();
+
+    expect(
+      screen.getByText(
+        /Your compliance obligation for the 2024 reporting year was/i,
+      ),
+    ).toBeVisible();
+    expect(
+      screen.getByText(/An automatic overdue penalty has been incurred/i),
+    ).toBeVisible();
+  });
```

Comment:
Same scenario? When we use the version of the first alert with 'was', I think the second alert always shows?